docs/content/preview/explore/ycql-language/keyspaces-tables.md

```diff
@@ -0,0 +1,177 @@
+ cassandra |      True |         True |          [] | $2a$12$64A8Vo0R3K9XeUp26CSzpuWtvUBwOiGFjPAbXGt7wsxZIScGrcsDu\x00\x00\x00\x00
+```
+
+## Keyspaces
+
+YCQL supports keyspaces, much like Apache Cassandra, with one caveat. Keyspaces in Apache Cassandra
+specify the replication properties for all tables in that keyspace. Replication configuration is defined at the [universe](../../../architecture/concepts/universe/) level in YugabyteDB.
+
+To create a new keyspace `testdb`, run the following statement:
+
+```sql
+ycqlsh> CREATE KEYSPACE testdb;
+```
+
+To list all keyspaces, use the following command:
+
+```sql
+ycqlsh> DESCRIBE KEYSPACES;
+```
+
+```output
+system_schema  system_auth  testdb  system
+```
+
+To use a specific keyspace, use the following command:
+
+```sql
+ycqlsh> USE testdb;
+```
+
+You should see the following output:
+
+```output
+ycqlsh:testdb>
+```
+
+To drop the keyspace you just created, use the `DROP` command as follows:
+
+```sql
+ycqlsh> DROP KEYSPACE testdb;
+```
+
+Verify the keyspace is no longer present as follows:
+
+```sql
+ycqlsh> DESCRIBE KEYSPACES;
+```
+
+```output
+system_schema  system_auth  system
+```
+
+## Tables
+
+Create a table using the CREATE TABLE statement.
+
+```sql
+CREATE TABLE users (
+  id UUID,
+  username TEXT,
+  enabled boolean,
+  PRIMARY KEY (id)
+  );
+```
+
+To list all tables, use the following command:
+
+```sql
+ycqlsh> DESCRIBE TABLES;
+```
+
+```output
+ycqlsh:testdb> DESCRIBE TABLES;
+
+users
+```
+
+To describe the table you created, enter the following:
+
+```sql
+ycqlsh> DESCRIBE TABLE users;
+```
+
+```output
+ycqlsh:testdb> describe table users;
+
+CREATE TABLE testdb.users (
+    id uuid PRIMARY KEY,
+    username text,
+    enabled boolean
+) WITH default_time_to_live = 0
+    AND transactions = {'enabled': 'false'};
+```
+
+{{< note title="Note" >}}
+Due to architectural differences, YugabyteDB does _not_ support most of the Apache Cassandra table level properties.
+`default_time_to_live` is one of the supported properties and the `transactions` property is added by YugabyteDB. For more details, see [table properties](../../../api/ycql/ddl_create_table/#table-properties-1).
+{{< /note >}}
+`default_time_to_live` is one of the supported properties. YugabyteDB adds the
+`transactions` property. See [table properties](/preview/api/ycql/ddl_create_table/#table-properties-1) for more details.
+
+## Quit ycqlsh
+
+To quit the shell, enter the following command:
+
+```sql
+ycqlsh> quit;
+```
+
+## Unsupported `cqlsh` commands
+
+|   Command         |                      Alternative                          |
+| :---------------- | :-------------------------------------------------------- |
+| LIST ROLES        | select * from system_auth.roles                           |
+| SHOW SESSION      | Tracing from `ycqlsh` is not supported.                   |
+| TRACING           | Tracing from `ycqlsh` is not supported.                   |
+{.sno-1}
```

Comment:
Could we add a "Learn More" section to point the user to other pages?

Reply:
Which other pages?

Reply:
Like Create Keyspace/ Create table API pages ?